src/lib.rs

```diff
@@ -394,6 +394,30 @@ where
     runner.run(f);
 }
 
+/// Sets the `tag` field of the current task.
+pub fn set_tag(tag: u64) {
+    use runtime::execution::ExecutionState;
+    ExecutionState::set_tag(tag);
+}
+
+/// Gets the `tag` field of the current task.
+pub fn get_tag() -> u64 {
+    use runtime::execution::ExecutionState;
+    ExecutionState::get_tag()
+}
+
+/// Gets the current step count.
+pub fn step_cnt() -> usize {
+    use runtime::execution::ExecutionState;
+    ExecutionState::step_cnt()
+}
+
+/// Gets the current step count.
```

Comment:
Copy-n-paste error in comment.

But in any case, I don't see why we are providing a public function to expose the current TaskId as a String.  If needed, we should just expose the `TaskId` itself.